poc/tests/test_vdaf_poplar1.py

```diff
@@ -285,6 +285,38 @@ def test_aggregation_parameter_encoding(self) -> None:
         self.assertEqual(want, cls.decode_agg_param(
             cls.encode_agg_param(want)))
 
+    def test_aggregation_parameter_encoding_clear_trailing_bits(self) -> None:
```

Comment:
@divergentdave These only cover the last trailing bit being set. Shoiuld we add some test for the last 7?

Reply:
Added more tests for this.